### PR TITLE
Document environment variable options for configuring .NET agent logging features

### PR DESCRIPTION
**Pull request by nr-ahemsath**
Update the new application logging feature config options for the .NET agent to include the environment variable config options.  Thanks to @Lonli-Lokli for pointing out this gap in our documentation.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 1]
```diff
@@ -3105,6 +3105,16 @@ For more details, see our documentation about [APM logs in context](/docs/apm/ne
 </applicationLogging>
 ```
 
+These features can also be configured via environment variables:
+
+```
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3110]: @nr-ahemsath curious if you think it makes sense to include the values for these (="true") or if it'd be better to leave the values off if just giving examples.

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 2]
```diff
@@ -3105,6 +3105,16 @@ For more details, see our documentation about [APM logs in context](/docs/apm/ne
...
+NEW_RELIC_APPLICATION_LOGGING_ENABLED="true"
+NEW_RELIC_APPLICATION_LOGGING_METRICS_ENABLED="true"
+NEW_RELIC_APPLICATION_LOGGING_FORWARDING_ENABLED="false"
+NEW_RELIC_APPLICATION_LOGGING_FORWARDING_MAX_SAMPLES_STORED="10000"
+NEW_RELIC_APPLICATION_LOGGING_LOCAL_DECORATING_ENABLED="false"
+```
+
 <Callout variant="important">
```
Comment (zuluecho9) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3118]: you placed your addition in between the thing above and this callout so I'm curious if you have an opinion about where that callout should live.

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx (hunk 1 above)
Comment (nr-ahemsath) [reply src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3110]: I was back and forth on it myself; in the end I decided to parallel the XML config file example shown immediately above the new section by having some concrete values.  I could go either way.

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx (hunk 2 above)
Comment (nr-ahemsath) [reply src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3118]: All of the config options under `<applicationLogging>` are only available in 9.7.0+, so at the very least we should change the wording to be "These configuration options are only available in the .NET agent v9.7.0 or higher."

I could see it going before the XML config examples.  If you agree I can make the changes myself.

Comment (zuluecho9) [reply src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3118]: I'll get it. Thanks!

**Review by zuluecho9 — APPROVED** [review]

**Pull request merged**